data/weapons.txt

```diff
@@ -777,10 +777,10 @@ outfit "Javelin Pod"
 		"reload" 15
 		"firing energy" .2
 		"firing heat" 12
-		"shield damage" 57
-		"hull damage" 36
+		"shield damage" 48
+		"hull damage" 26
```

Comment:
I have to agree with @Nescio0 here: with these changes, there's very little reason to use Javelins anymore. If the damage needs to be reduced, I would recommend doing it by not so much-- or not doing it at all.

Reply:
As in my reply to him, if the only way to make the Javelin viable over the Meter is to give the Javelin it’s 68% DPS buff, then the Javelin was too weak to begin with. Given that the Javelin being buffed that much caused complaints (and honestly I think it’s too much given that it makes a 12 ton weapon the highest DPS human weapon outside of nukes), I’d rather go about reducing Meteor damage.

Reply:
>  then the Javelin was too weak to begin with

I think it certainly was.

> I’d rather go about reducing Meteor damage.

Especially with reduced frequency, I don't think Meteor damage will be a problem.